src/App.js

```diff
@@ -87,20 +123,39 @@ const App = () => {
     <div className={classes.root}>
       <Container>
         {!image ? (
-          <div className={classes.faceCapture}>
-            <FaceCapture
-              captureMethod={CAPTURE_METHOD.AUTO}
-              onSuccess={onSuccess}
-              onError={onError}
-            />
+          <div className={classes.scanContainer}>
+            <div className={classes.faceCaptureWrapper}>
+              <FaceCapture
+                captureMethod={CAPTURE_METHOD.AUTO}
```

Comment:
I think AUTO is the default mode we could remove it, no?

Reply:
You are right, didn't thought about that :)

Reply:
Done